lib/coverband/utils/source_file.rb

```diff
@@ -103,6 +104,13 @@ def initialize(filename, file_data)
         end
       end
 
+      def runtime_relavant_calculations(runtime_relavant_lines)
```

Comment:
Ok. Seems like the reason for the proc is that temporarily while this method is being called, relevant_lines behavior changes to return the instance variable? I am a little confused still because it does not seem to be passing in a proc over [here](https://github.com/danmayer/coverband/pull/270/files#diff-db957067981a31c0b4a7f4bcc0f8843aR42).

Reply:
Oh I see. The erb is invoking the `result#runtime_relevant_coverage` since it does not take a proc.